custom/icds_reports/reports/cas_reach_data.py

```diff
@@ -54,7 +54,6 @@ def get_data_for_daily_usage(date, filters):
     del config['prev_month']
 
     awc_this_month_data = get_data_for_awc_monthly(current_month, config)
-    awc_prev_month_data = get_data_for_awc_monthly(previous_month, config)
```

Comment:
this seems like a bigger performance benefit than the view change. could even be merged separately

Reply:
Sure I will send other Pr for this. To judge the performance benefit from not using the views.